src/Result.cpp

```diff
@@ -113,6 +122,11 @@ double Result::getOpsPerSecond() const
 	return 1.0 / (this->getUsPerCall() * 1.0e-6);
 }
 
+double Result::getMBPerSecond() const
```

Comment:
Perhaps a more general approach here would be "getUnitsPerSecond" or something like that?  What you are really doing is interpreting the results based on the problem space size.  In your case, the "problem space" is megabytes.  It could, however, be any arbitrary unit I would suppose.